core-shaded/pom.xml

```diff
@@ -303,6 +303,7 @@
             </goals>
             <configuration>
               <instructions>
+                <Automatic-Module-Name>com.datastax.oss.driver.core</Automatic-Module-Name>
```

Comment:
For the shaded core, we have to do this from the bundle plugin, because we generate a separate manifest file and include it manually in the final archive.